### DISCUSSION TITLE
Typescript error in Getting started example

### DISCUSSION THREAD

**Discussion by mass2527** [nick-keller/react-datasheet-grid#158] [Q&A] [answered]
Hello nick-keller!

I want to use react-datasheet-grid with my react-typescript project 
since it has imposing features with great documentation.

But I think there's an issue when it's used with typescript.

here's the thing 
When I copy-paste this [Getting started example](https://react-datasheet-grid.netlify.app/docs/getting-started#usage),
typescript complains since `value` is inferred as
```
Record<string, any>[] 
```
However type of data is this
```
{
    active: boolean;
    firstName: string;
    lastName: string;
}
```

<img width="606" alt="스크린샷 2022-05-29 오후 12 13 56" src="https://user-images.githubusercontent.com/70563791/170850403-9fe7bdb8-08b6-4dab-bca4-b4df19076f8c.png">

Maybe the issue above is caused by `keyColumn` function's incorrect type?!
(typescript stopped blaming the wrong type when I removed `keyColumn` from the above example because data was correctly inferred)

Could you please check this issue and give me some feedback?!
I'll wait for your answer. thanks for reading.

**Comment by nick-keller** [comment, marked as answer]
Hey @mass2527 ,
I just added a [TS section](http://localhost:3000/docs/typescript) in the doc, and some TS examples throughout, it should be easier now ;)
